contributing-guides/style-guide.md

```diff
@@ -91,3 +92,27 @@ The example above does not use a serial comma, so this could mean one of two thi
 This can be resolved by inserting a comma before the "and" or "or" in the final element in the list.
 
 > Delete the Git branches, tags, and remotes.
+
+## Chinese-Specific Rules
+
+When Chinese words, Latin words and Arabic numerals are written in the same sentence, it takes more attention to copywriting.
+
+The following guidelines are applied to Chinese (zh) and traditional Chinese (zh_TW):
+
+1. Place one space before/after English words.  
+   For example, use `列出所有 docker 容器` rather than `列出所有docker容器`.
+2. Place one space before/after numbers.  
+   For example, use `宽度为 50 个字` rather than `宽度为50个字`.
+3. Place one space between numbers and units **except** degrees and percentages.  
+   For example, use `容量 50 MB` rather than `容量 50MB`.  
+   For instances of degree and percentage, use `50°C` and `50%` rather than `50 °C` and `50 %`.
```

Comment:
I wouldn't say 50MB is really wrong. Also, how does that only apply to Chinese?

Reply:
It depends on an organization's decision. Internet giants like Google, Microsoft, Wikipedia apply this rule in Chinese regions. But Apple doesn't.

Reply:
I'm still not sure if we should consider this Chinese-specific.

Reply:
Okay, I cancel this rule. If there would be a unit-related issue, this could be discuss again.